### PR TITLE
Depend on bulma from npm, rather than vendoring it.

### PR DESCRIPTION
**Pull request by tomprince**

### CODE REVIEW

File: src/bulmaOverride.css [hunk 1]
```diff
@@ -53,3 +53,11 @@ h6 {
...
   font-family: "Lato";
 }
+
+.subtitle.is-6 {
+	margin-bottom: 0.25rem;
+}
+
+.title:not(.is-spaced)+.subtitle {
+	margin-top: -1rem;
+}
```
Comment (tomprince) [inline src/bulmaOverride.css:63]: The rule appears to be unused, so I'd be inclined to remove it.

**Comment by tomprince** [comment]
While I was working on something else, I discovered that bulma does not have good accessibility support (notably, a lack of indicator for when most buttons are focused), so I'd be inclined to switch away from bulma. That being said, I still think this is worth merging in the mean time.

(ping @neubee)

**Comment by neubee** [comment]
I would be fine with changing to a different CSS library if someone wants to put the time into it. I'd lean towards tailwind or bootstrap, but I don't don't do much with CSS so feel free to use whatever you think is best.

**Comment by tomprince** [comment]
I've got three or four different branches, all mid CSS rewrite, each with a different approach. At some point,  I might actually finish one of them. 🙃

**Pull request merged**